homeassistant/components/google_assistant/trait.py

```diff
@@ -675,3 +676,77 @@ def can_execute(self, command, params):
         await self.hass.services.async_call(lock.DOMAIN, service, {
             ATTR_ENTITY_ID: self.state.entity_id
         }, blocking=True)
+
+
+@register_trait
+class FanSpeedTrait(_Trait):
+    """Trait to control speed of Fan.
+
+    https://developers.google.com/actions/smarthome/traits/fanspeed
+    """
+
+    name = TRAIT_FANSPEED
+    commands = [
+        COMMAND_FANSPEED
+    ]
+
+    speed_synonyms = {
+        "off": ['stop', 'off'],
```

Comment:
Can you use the `SPEED_` constants as the keys for this dictionary? That way it's clear to the reader where this info comes from.

Reply:
Done. Fixed in 60e677a921f7f5ca4b38e86caddf2a0590d9be84